lib/sepa.test.js

```diff
@@ -30,3 +36,135 @@ describe('IBAN tests',
       expect(SEPA.validateIBAN('DE1Zsantander')).toBe(false);
     });
   });
+
+describe('xml generation for transfer documents', () => {
+  const PAIN_FOR_TRANSFERS = 'pain.001.003.03';
+
+  function validTransferDocument({debtorId=A_VALID_CREDITOR_ID, debtorName='debtor-name'}) {
```

Comment:
(see my comment in the other thread https://github.com/kewisch/sepa.js/pull/60/files/52136f89dd003d19e9f1836a733561d5ca9651e5#r1406241685)

I pass in explicit values to underline the test fixture and make it more readable. But I guess that readers might also wonder why I am passing in the default arguments 😄 

In any case, let's continue the conversation in the other thread, since this is the same issue.